scancodeio/static/add-inputs.js

```diff
@@ -21,7 +21,42 @@
 // Visit https://github.com/nexB/scancode.io for support and download.
 
 const fileInput = document.querySelector('#id_input_files');
-fileInput.onchange = () => {
+fileInput.onchange = update_files
```

Comment:
Let's add a ";" at the end of the line for consistency with the rest of the code